conf/machine/xenclient-uivm.conf

```diff
@@ -9,3 +9,8 @@ require xenclient-common.conf
 MACHINE_FEATURES = "screen keyboard ethernet pci acpi ext2 x86"
 
 USE_VT = "0"
+
+module_conf_xen-fbfront = "blacklist xen-fbfront"
+KERNEL_MODULE_PROBECONF += " \
+    xen-fbfront \
```

Comment:
I think this can go away if you set vfb to false in (servicevm-uivm)[https://github.com/OpenXT/manager/blob/master/templates/default/service-uivm#L70].  That stops libxl from creating the xenstore entries which autoload xen-fbfront.  Doesn't hurt to keep this blacklisting, I suppose, but toggling off vfb should be done as well.

Reply:
Yes this is much simpler.